spacify-ui/package-lock.json

```diff
@@ -0,0 +1,17021 @@
+{
+  "name": "spacify-ui",
+  "version": "0.1.0",
+  "lockfileVersion": 3,
```

Comment:
let's not commit package lock anywhere. It is an auto generated file

Reply:
Okay. Unstaged it.